params/protocol_params.go

```diff
@@ -168,3 +190,29 @@ func (t CodeFormat) Validate() bool {
 	}
 	return false
 }
+
+func (t CodeFormat) GenerateCodeInfo(r Rules) CodeInfo {
+	return CodeInfo(t) | CodeInfo(GenerateVmVersion(r))
+}
```

Comment:
How about changing the function name and add a function like the following?
```suggestion
func (t CodeFormat) CodeInfoFromRules(r Rules) CodeInfo {
	return CodeInfo(t) | CodeInfo(GenerateVmVersion(r))
}
func CodeInfoFromCodeFormat(c CodeFormat) CodeInfo {
	return CodeInfo(c)
}
func CodeInfoFromVmVersion(c CodeFormat) CodeInfo {
	return CodeInfo(c)
}
```
We can set like `AccountValueKeyCodeInfo:      params.CodeInfoFromCodeFormat(params.CodeFormatEVM),` in account_test.go
and `sca.codeInfo = params.CodeInfoFromCodeFormat(serialized.CodeFormat) | params.CodeInfoFromVmVersion(serialized.VmVersion)` in smart_contract_account.go

Reply:
Forgot to response at this comment.. I reflected at https://github.com/klaytn/klaytn/pull/912/commits/afadf16289832acbd0491d966c9253ddbd8f8861!